C/plugins/storage/postgres/connection.cpp

```diff
@@ -3312,3 +3312,422 @@ const char *p;
+                                                        {
+								std::string s;
+								for (auto& i : v["index"].GetArray())
+								{
+									s.append(i.GetString());
+									s.append(",");
+								}
+								// remove last comma
+								if ( s[s.size()-1] == ',')
+								{
+									s.erase(s.size()-1);
+								}
+
+                                                               	indexSet.insert(s);
+                                                        }
+                                                }
+                                        }
+                                }
+
+				tableColumnMap[name] = columnSet;
+				tableIndexMap[name] = indexSet;
+
+			}
+		}
+
+	}
+
+	return true;
+}
+/**
+ * Create scema of tables
+ *
+ * @param payload   The  payload containing information about schema of 
+ *                  tables to create
+ * @return true if the tables can be crated successfully
+ */
+int Connection::create_schema(std::string payload)
+{
+	Logger::getLogger()->error ( "%s:%d payload is  %s", __FUNCTION__, __LINE__, payload.c_str());
+
+	Document document;
+	std::string schema;
+	int version;
+	const char *logSection="CreatingSchema";
+	unsigned long rowsAffectedLastCommand = 0;
+	std::unordered_map<std::string, unordered_set<std::string> > columnMap;
+        std::unordered_map<std::string, unordered_set<std::string> > indexMap;
+
+	bool schemaUpdateRequired = false;
+
+	try {
+                if (payload.empty())
+                {
+                        return -1;
+                }
+                else
+                {
+                        if (document.Parse(payload.c_str()).HasParseError())
+                        {
+                                raiseError("retrieve", "Failed to parse JSON payload");
+                                return -1;
+                        }
+			if (!document.HasMember("schema"))
+                        {
+				Logger::getLogger()->error("schema absent from payload, so exiting ");
+				return -1;
+			}
+			else
+			{
+				schema = document["schema"].GetString();
+				if (!document.HasMember("service"))
+				{
+					Logger::getLogger()->debug("service absent from payload, so exiting");
+                                        return -1;
+				}
+				std::string service = document["service"].GetString();	
+				if (service.empty())
+				{
+					Logger::getLogger()->debug("empty service name, so exiting");
+                                        return -1;
+				}
+
+				if (!document.HasMember("version"))
+                        	{
+					Logger::getLogger()->debug("version absent from payload, so exiting");
+                                	return -1;
+                        	}
+				else
+				{
+					version = document["version"].GetInt();
+
+					// insert payload in 
+			//std::string s = "insert into fledge.service_schema(name, version, definition) values ('" + schema + "', " + to_string(version) + ", " + "'" + payload + "') ;" ;
```

Comment:
If this code is not needed any more then please remove it, if it is there for another reason then explain why in the comment so that future developers understand its purpose rather than think it has accidentally been commented out.